src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemCsv.scala

```diff
@@ -0,0 +1,51 @@
+package nl.knaw.dans.easy.stage.fileitem
```

Comment:
As far as I can see what you are trying to achieve is:
1. Take an input CSV `File`
2. Parse and validate it (required headers present? no invalid headers?)
3. Return a collection of `FileItemSetting`s to feed to the file item staging code.

That is fine. I see the following main problems with this object:

* I think it is a bad idea to use the command line as the interface to the file item staging code. The command line is very clear when you use it *from the command line* but if you assemble a command line in code it becomes obscure. So, you should not do that unless there is no other way (for instance when the command line application is not part of your own code base).

* This object is not clear about the validity of an empty CVS-file. It logs a warning an then returns `Success`.  Remember the part in the Scala workshop about "being honest"? It refers to type signatures conveying what the code does.  `read` is the "API" you are providing to the upper layer here. Its type signature should tell what the possible results are so that the caller can decide what the appropriate action is:
  * Can `read` return "nothing" (without it being an error per se)? - Then use `Option` in the return type
  * Can `read` fail? - Then use `Try` in the return type
  * Are both possible? - Then use `Try[Option]`
  
  So, if an empty or absent CSV-file is considered an error (seems reasonable to me) you can just use a `Try`. Logging this can be done by the caller. What about the warnings about invalid headers? You should really consider whether they are not actually errors. But if you insist that they should only be warnings you could make the list of ignored headers part of the return type, e.g. 

       type IgnoredHeaders = Seq[String]
       // ...
       Try[(FileItemSettings, IgnoredHeaders)]